### PR TITLE
Update documentation for theme and set-goal

### LINKED ISSUE #88: Fix `set-goal` command formatting
Screenshot 2021-03-28 at 10 40 42

Rendered on website

### PR DESCRIPTION
**Pull request by lue97**
Fixes #88

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -260,10 +260,10 @@ See also:
 
 ### Setting meeting goal: `set-goal`
 
-Format: <code>set-goal INDEX f/w[eek[ly]] &vert; m[onth[ly]] &vert; y[ear[ly]] &vert; n[one]</code>
+Format: <code>set-goal INDEX f/FREQUENCY</code>
```
Comment (ivantjh) [inline docs/UserGuide.md]: Format: `set-goal INDEX f/FREQUENCY` can just use back ticks.

File: docs/diagrams/ThemeCommandSequenceDiagram.puml [hunk 2]
```diff
@@ -33,10 +33,6 @@ ThemeFactory --> ThemeCommand : theme
 
 ref over ThemeCommand, ThemeFactory : ThemeManager#setTheme(theme, themePath)
 
-ThemeCommand -> ThemeManager : applyThemeToScene()
```
Comment (ivantjh) [inline docs/diagrams/ThemeCommandSequenceDiagram.puml:36]: Where is the new `ThemeCommandSequenceDiagram.png`?

**Review by ivantjh — CHANGES_REQUESTED** [review]

File: docs/diagrams/ThemeCommandSequenceDiagram.puml (hunk 2 above)
Comment (lue97) [reply docs/diagrams/ThemeCommandSequenceDiagram.puml:36]: It's updated as a binary file with the same file name.

Comment (ivantjh) [reply docs/diagrams/ThemeCommandSequenceDiagram.puml:36]: Oops, my bad.

**Review by ivantjh — APPROVED** [review]

**Review by chesterhow — APPROVED** [review]
LGTM, maybe can give more examples for `set-goal`?

**Pull request merged**